tests/test_image/test_io.py

```diff
@@ -9,11 +9,15 @@
 import cv2
 import numpy as np
 import pytest
+import torch
 from numpy.testing import assert_allclose, assert_array_equal
 
 import mmcv
 from mmcv.fileio.file_client import HTTPBackend, PetrelBackend
 
+if torch.__version__ == 'parrots':
+    pytest.skip('not necessary in parrots test', allow_module_level=True)
```

Comment:
Why do we need to skip those unit tests?

Reply:
Some libs is not installed in parrots docker images, and it is not necessary for parrots test.

Reply:
Got it